src/lib/utils/get-token.utils.ts

```
@@ -0,0 +1,30 @@
+
+    if (!raw) {
+        return null;
+    }
+
+    const secret = process.env.NEXTAUTH_SECRET;
+    if (!secret) {
+        return null;
+    }
+
+    try {
+        const verified: any = jwt.verify(raw, secret);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)